src/seedu/addressbook/data/person/Address.java

```diff
@@ -8,10 +8,22 @@
  */
 public class Address {
 
-    public static final String EXAMPLE = "123, some street";
+    public static final String EXAMPLE = "123, some street, #21-03, 530161";
+    // Format of address input: a/BLOCK, STREET, UNIT, POSTAL_CODE
     public static final String MESSAGE_ADDRESS_CONSTRAINTS = "Person addresses can be in any format";
     public static final String ADDRESS_VALIDATION_REGEX = ".+";
-
+    public static final String SEPARATION = ",";
+    
+    private AddressBlock block;
+    private AddressStreet street;
+    private AddressUnit unit;
+    private AddressPostalCode postalCode;
+    
+    public static final int BLOCK_IDX = 0;
```

Comment:
Good use of constant to avoid magic number. However, try not to use uncommon abbreviation like IDX.